### PR TITLE
Update Stylelint to 16.10.0

### PR DESCRIPTION
**Pull request by MariaSolOs**
#### Changes proposed in this pull request:

In an effort of keeping dev dependencies up to date, fixing the setup here to use the latest Stylelint version.

#### Reviewers should focus on:

N/A? I guess that if CI is happy we should be okay 😸

### CODE REVIEW

File: packages/core/src/common/_mixins.scss [hunk 1]
```diff
@@ -38,11 +38,8 @@ $pt-dark-intent-text-colors: (
 }
 
 @mixin position-all($position, $value) {
-  bottom: $value;
-  left: $value;
+  inset: $value $value $value $value;
```
Comment (ggdouglas) [inline packages/core/src/common/_mixins.scss]: **nit:** can we write this as:
```suggestion
  inset: $value;
```

File: packages/table-dev-app/src/index.scss [hunk 2]
```diff
@@ -24,11 +21,8 @@ body {
 
 /* stylelint-disable-next-line selector-max-id */
 #page-content {
-  bottom: 0;
-  left: 0;
+  inset: $pt-navbar-height 0 0 0;
```
Comment (ggdouglas) [inline packages/table-dev-app/src/index.scss]: **nit:** can we write this as:
```suggestion
  inset: $pt-navbar-height 0 0;
```

File: packages/table/src/headers/_headers.scss [hunk 3]
```diff
@@ -294,10 +294,8 @@ $dark-selectable-header-menu-selected-hover-background: menu-background(
   &.#{$ns}-editable-text {
     &::before {
       border-radius: 0;
-      bottom: -$cell-padding-vertical;
-      left: -$cell-padding-horizontal + $cell-border-width - 2px;
-      right: -$cell-padding-horizontal + $cell-border-width - 1px;
-      top: -$cell-padding-vertical + $cell-border-width - 2px;
+      inset: -$cell-padding-vertical + $cell-border-width - 2px -$cell-padding-horizontal + $cell-border-width -
+        1px -$cell-padding-vertical -$cell-padding-horizontal + $cell-border-width - 2px;
```
Comment (ggdouglas) [inline packages/table/src/headers/_headers.scss]: Is the value of this `inset` property equivalent to the above?

Also, the formatted result here is a bit harder to read vs. the original. Is there any way we can break this back out into multiple lines?

Comment (MariaSolOs) [reply packages/table/src/headers/_headers.scss]: We can keep the original and just disable the rule that prefers a single `inset` property.

**Review by ggdouglas — APPROVED** [review]
Thanks for tackling this! 🙂

**Comment by MariaSolOs** [comment]
> Thanks for tackling this! 🙂

Thank you for the review and welcoming the change!

**Pull request merged**